brooklyn-server/server-cli/src/main/java/org/apache/brooklyn/cli/ItemLister.java

```diff
@@ -198,17 +277,36 @@ private void mkdir(String rootDir, String dirName) {
 
         protected List<URL> getUrls() throws MalformedURLException {
             List<URL> urls = Lists.newArrayList();
-            if (jars.isEmpty()) {
-                String classpath = System.getenv("INITIAL_CLASSPATH");
-                if (Strings.isNonBlank(classpath)) {
-                    List<String> entries = Splitter.on(":").omitEmptyStrings().trimResults().splitToList(classpath);
-                    for (String entry : entries) {
-                        if (entry.endsWith(".jar") || entry.endsWith("/*")) {
-                            urls.addAll(ClassFinder.toJarUrls(entry.replace("/*", "")));
+            if (jars.isEmpty()) { //TODO: where is `jars` value write - AbstractMain.execCli(Cli<BrooklynCommand> parser, String ...args):line 250 BrooklynCommand command = parser.parse(args)
```

Comment:
Yes, could print an error and exit, but I'm fine with current behaviour. What I was referring to though is the "TODO" note on the right.